azero/contracts/shared/src/ownable2step.rs

```diff
@@ -0,0 +1,64 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
```

Comment:
Seems like this code isn't used currently, am I wrong?

Reply:
Right. It comes from the initial commit. It shall be removed before the merge.